automatic/vscode/tools/helpers.ps1

```diff
@@ -14,4 +14,8 @@ function Close-VSCode {
     Get-Process code -ea 0 | ForEach-Object { $_.CloseMainWindow() | Out-Null }
     Start-Sleep 1
     Get-Process code -ea 0 | Stop-Process  #in case gracefull shutdown did not succeed, try hard kill
-}
\ No newline at end of file
+}
+
+function Get-VSCodeVersion {
+  return (Get-UninstallRegistryKey 'Microsoft Visual Studio Code').DisplayVersion
```

Comment:
It would be nice to have the software name only on a single location. It is already set in [ChocolateyInstall.ps1](https://github.com/casz/chocolatey-coreteampackages/blob/28949e71e6d208c210684dc5ffe1c86338f3372e/automatic/vscode/tools/ChocolateyInstall.ps1#L21). I would suggest to move this code to `ChocolateyInstall.ps1` as it anyway is only a single line of code and put the software name in a variable there.